tests/test_cluster/test_silhouette.py

```diff
@@ -183,7 +183,7 @@ def test_quick_method(self):
         )
 
         model = MiniBatchKMeans(3, random_state=343)
-        oz = silhouette_visualizer(model, X, random_state=93, legend=False)
+        oz = silhouette_visualizer(model, X, show=False)
```

Comment:
:+1:

Reply:
@bbengfort - could you explain what you mean by `Models` and `Workflow` in your example above?

Reply:
Hey there @pdamodaran! Hope things have been well! Thought I'd jump in here since I couldn't resist 🙂 

`Models`: The type of machine learning that the `Visualizer` is designed for (e.g. Classification, Regression, Clustering &mdash; akin to the "default task" categorization on the [UCI Machine Learning Repository](https://archive.ics.uci.edu/ml/datasets.php)). For example, what kind of machine learning should you expect to do with a `SilhouetteVisualizer`?

`Workflow`: Where does this `Visualizer` fit into the [machine learning workflow](https://www.scikit-yb.org/en/develop/about.html#model-selection) &mdash; is it used for feature analysis, model evaluation, or hyperparameter tuning? For instance, at what stage of the workflow would a `ClassificationReport` be used?

Reply:
Thanks @rebeccabilbro for the explanation - it makes sense to me now.

@bbengfort - The template you suggested above looks good to me.